model/src/main/java/com/basistech/rosette/apimodel/TextEmbeddingResponse.java

```diff
@@ -33,7 +33,17 @@
 public class TextEmbeddingResponse extends Response {
 
     /**
-     * @return the embedding vector as a list
+     * @return the document embedding vector as a list
      */
-    private final List<Double> embedding;
+    private final List<Double> documentEmbedding;
```

Comment:
Aren't we deprecating the old one?

Reply:
I'm double-checking with Hannah to be sure, but I think the plan is to just make the change (since TVEC is still in "Labs")

Reply:
Alright, Hannah just confirmed that we can go ahead and just rename the field. She will make sure that the appropriate customers are notified about the change.